src/main/java/DefenitlyNotAJoikedInstallerFrame.java

```diff
@@ -0,0 +1,682 @@
+            } catch (Throwable ivjExc) {
+                showErrorPopup(ivjExc);
+            }
+        }
+        return buttonOpenFolder;
+    }
+
+    private JButton getButtonClose() {
+        if (buttonClose == null) {
+            w = 100;
+            h = 26;
+
+            try {
+                (buttonClose = new JButton()).setName("ButtonClose");
+                buttonClose.setPreferredSize(new Dimension(w, h));
+                buttonClose.setText("Cancel");
+            } catch (Throwable ivjExc) {
+                showErrorPopup(ivjExc);
+            }
+        }
+        return buttonClose;
+    }
+
+    public void onFolderSelect() {
+        File currentDirectory = new File(getFieldFolder().getText());
+
+        JFileChooser jFileChooser = new JFileChooser(currentDirectory);
+        jFileChooser.setFileSelectionMode(JFileChooser.DIRECTORIES_ONLY);
+        jFileChooser.setAcceptAllFileFilterUsed(false);
+        if (jFileChooser.showOpenDialog(this) == 0) {
+            File newDirectory = jFileChooser.getSelectedFile();
+            getFieldFolder().setText(newDirectory.getPath());
+        }
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        if (e.getSource() == getButtonClose()) {
+            dispose();
+            System.exit(0);
+        }
+        if (e.getSource() == getButtonFolder()) {
+            onFolderSelect();
+        }
+        if (e.getSource() == getButtonInstall()) {
+            onInstall();
+        }
+        if (e.getSource() == getButtonOpenFolder()) {
+            onOpenFolder();
+        }
+    }
+
+    @Override
+    public void mouseClicked(MouseEvent e) {
+        if (e.getSource() == getForgeTextArea()) {
+            try {
+                Desktop.getDesktop().browse(new URI("http://files.minecraftforge.net/maven/net/minecraftforge/forge/index_1.8.9.html"));
+            } catch (IOException | URISyntaxException ex) {
+                showErrorPopup(ex);
+            }
+        }
+    }
+
+    public void onInstall() {
+        try {
+            File modsFolder = new File(getFieldFolder().getText());
+            if (!modsFolder.exists()) {
+                showErrorMessage("Folder not found: " + modsFolder.getPath());
+                return;
+            }
+            if (!modsFolder.isDirectory()) {
+                showErrorMessage("Not a folder: " + modsFolder.getPath());
+                return;
+            }
+            tryInstall(modsFolder);
+        } catch (Exception e) {
+            showErrorPopup(e);
+        }
+    }
+
+    private void tryInstall(File modsFolder) {
+        File thisFile = getThisFile();
+
+        if (thisFile != null) {
+            boolean inSubFolder = false;
+            if (IN_MODS_SUBFOLDER.matcher(modsFolder.getPath()).find()) {
+                inSubFolder = true;
+            }
+
+            File newFile = new File(modsFolder, "Danker's Skyblock Mod-"+getVersionFromMcmodInfo()+".jar");
```

Comment:
Not sure if this will work from github, as the file name is different

Reply:
If you build it and use the file name you get it works

Reply:
fixed it by changing the it to `File newFile = new File(modsFolder, getVersionFromMcmodInfo() + ".Danker.s.Skyblock.Mod.-." + DankersSkyblockMod.VERSION + ".jar");`